precompiles/staking/method_get_validators.go

```diff
@@ -0,0 +1,64 @@
+package staking
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	distrkeeper "github.com/cosmos/cosmos-sdk/x/distribution/keeper"
+	distrtypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+
+	precompiletypes "github.com/zeta-chain/node/precompiles/types"
+)
+
+// getValidators queries the list of validators for a given delegator.
+func (c *Contract) getDelegatorValidators(
```

Comment:
I feel like `getValidatorListForDelegatorAddress` might be clearer, but I don't feel strongly about it, though

Reply:
`getValidatorListForDelegator` sounds good.